### PR TITLE
Add book title and author to profile history

### PR DESCRIPTION
**Pull request by deltaisfordeath**

### CODE REVIEW

File: src/GRA.Controllers/HomeController.cs [hunk 1]
```diff
@@ -222,6 +223,16 @@ var schedulingStage
                         .GetUpcomingStreamListAsync()
                 };
 
+                var latestBookCookie = Request.Cookies[$"{GetActiveUserId()}{LatestBook}"];
+                if (latestBookCookie != null)
+                {
+                    var title = latestBookCookie.Split(";")[0];
```
Comment (k7hpn) [inline src/GRA.Controllers/HomeController.cs:229]: Did you test this if the string doesn't contain `;`? I suspect it reacts....poorly. Maybe check if `.Contains(";")` and if not assume it's title only?

Comment (k7hpn) [reply src/GRA.Controllers/HomeController.cs:229]: Oh I see below that you always put a `;` even if there is no author. I'd still add the `.Contains()` here as it's an exception if you split on a character that's not present I think.

Comment (deltaisfordeath) [reply src/GRA.Controllers/HomeController.cs:229]: I just tested it and Split returns an array of size 1 containing the original string if the split character is not present.

Comment (deltaisfordeath) [reply src/GRA.Controllers/HomeController.cs]: Just pushed a fix. If you split on a character that isn't present the return is an array of length 1 with the original string, but that would still have caused an error when trying to access the Author from cookie[1].

**Pull request merged**